src/javascript/JContent/EditFrame/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -26,39 +26,24 @@ export const Breadcrumbs = ({nodes, setCurrentElement, onSelect, isResponsiveMod
         return false;
     }, [onSelect, selection.length, setCurrentElement]);
 
-    if (isResponsiveMode) {
-        const data = nodes.map(n => ({
-            label: n.name,
-            value: n.path,
-            image: <NodeIcon node={n}/>
-        }));
-
-        return (
-            <Dropdown data={data}
-                      placeholder={t('jcontent:label.contentManager.pageBuilder.breadcrumbs.dropdownLabel')}
-                      onChange={(e, v) => {
-                          handleItemOnClick(e, v.value);
-                      }}
-            />
-        );
-    }
+    const data = nodes.toReversed().map((n, index) => ({
+        label: n.name,
+        value: n.path,
+        image: <NodeIcon node={n} style={{marginLeft: 8 * index}}/>
+    }));
```

Comment:
Can I ask why the change of styling here? Looks ok to me atm: 
![image](https://github.com/user-attachments/assets/38d4a297-35b5-496f-9f78-d9597d1a0cf2)

Reply:
Clement asked for this to better show hierarchy: 
<img width="178" alt="image" src="https://github.com/user-attachments/assets/7a36931a-5145-4428-a12d-9a5a4435dbd1" />

Reply:
<img width="1009" alt="image" src="https://github.com/user-attachments/assets/d81d3311-2a50-4f13-a331-3ea6e7a6514b" />

Reply:
Ah, got it, I missed this requirement. Thanks for clarifying.